src/server_manager/web_app/shadowbox_server.ts

```diff
@@ -129,12 +130,21 @@ export class ShadowboxServer implements server.Server {
     return new Date(this.serverConfig.createdTimestampMs);
   }
 
+  async setHostname(hostname: string): Promise<void> {
```

Comment:
We should be consistent with naming. I see `setHostname`, `getHostname`, `hostname-for-access-keys` and `hostnameForAccessKeys`. They should all match.

Reply:
changed all to *ForAccessKeys